server/src/main/java/org/opensearch/cluster/routing/OperationRouting.java

```diff
@@ -83,10 +83,18 @@ public class OperationRouting {
         Setting.Property.Dynamic,
         Setting.Property.NodeScope
     );
+
+    public static final Setting<Boolean> WEIGHTED_ROUTING_FAILOPEN_ENABLED = Setting.boolSetting(
+        "cluster.routing.weighted.fail_open",
```

Comment:
should we enable it by default as it is already experimental ?

Reply:
We can do it